utility/src/main/java/org/oppia/android/util/profile/ProfileNameValidator.kt

```diff
@@ -35,6 +37,16 @@ class ProfileNameValidator @Inject constructor() {
     return !name.contains(noRepeatedAllowedSymbolsRegex)
   }
 
+  /** Checks if name contains only symbols */
+  private fun doesNotContainOnlyAllowedSymbolsAndSpaces(name: String): Boolean {
+    name.forEach {
+      if (it.isAlphabetic()) {
+        return true
+      }
+    }
+    return false
+  }
```

Comment:
Can be simplified to:
```suggestion
 private fun doesNotContainOnlyAllowedSymbolsAndSpaces(name: String): Boolean {
    return name.any { it.isAlphabetic() }
  }
```

Reply:
Thanks @adhiamboperes , will change it. I used `.forEach` because it was previously used in the same file and I just used that as a reference.
While I am at it 
```
  private fun containsOnlyLettersAndAllowedSymbols(name: String): Boolean {
    name.forEach {
      if (!(it.isAlphabetic() || it in repeatableSymbols)) {
        return false
      }
    }
    return true
  }
```
can also be simplified to 
```
 private fun containsOnlyLettersAndAllowedSymbols(name: String): Boolean {
    return name.all{ it.isAlphabetic() || it in repeatableSymbols }
}
```
should i change that aswell?

Reply:
Yes, that would be fine.

Reply:
done